tests/voq/test_voq_fabric_isolation.py

```diff
@@ -0,0 +1,104 @@
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.utilities import wait_until
+import logging
+import pytest
+import random
+
+logger = logging.getLogger(__name__)
+
+# This test only runs on t2 systems.
+pytestmark = [
+    pytest.mark.topology('t2')
+]
+
+# This test checks if fabric link monitoring algorithm works as expected.
+# To do this, the test updates fake errors ( crc errors ) on a
+# fabric link first, then check if the auto_isolated state get updated in
+# state_db for this link. Then it clears the fake errors on that link and
+# check if the auto_siolated state get cleared as well.
+
+
+def test_voq_fabric_isolation_status(duthosts, enum_frontend_dut_hostname):
+    """Check if the fabric link isolation status updated correctly by
+    the link monitoring algorithm"""
+
+    logger.info("Checking fabric serdes link status")
+
+    # Get a link
+    duthost = duthosts[enum_frontend_dut_hostname]
+    logger.info("Testing on duthost: {}".format(duthost.hostname))
+
+    port = random.randint(0, 111)
```

Comment:
is the port range platform specific, why the port range is till 111?

Reply:
the smallest number of total fabric links on supported card  is 111, so I just randomly choose one port to check